code/game/gamemodes/objectives/_objective.dm

```diff
@@ -126,10 +126,10 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 
 /datum/objective/proc/get_crewmember_minds()
 	. = list()
-	for(var/datum/data/record/R as() in GLOB.data_core.locked)
-		var/datum/mind/M = R.fields["mindref"]
-		if(M)
-			. += M
+	for(var/datum/record/locked/target in GLOB.manifest.locked)
+		var/datum/mind/mind = target.mind_ref
```

Comment:
this has some issue of harddel or something, but I will give you exact change once other things are sorted out first. Leaving a reminder note here for now.